File: ShopFR/src/app/model/ordini.ts
export class Ordini {

    private id_user !: number;
    private id_prodotti !: number;

    constructor(id_user: number, id_prodotti: number) {
        this.id_user = id_user;
        this.id_prodotti = id_prodotti;
    }

    getId_user(): number {
        return this.id_user;
    }

    getId_prodotti(): number {        
        return this.id_prodotti;
    }

}

File: ShopBK/src/main/java/com/example/shopbk/Model/Ordini.java
package com.example.shopbk.Model;

import jakarta.persistence.*;
import lombok.Data;

@Data
@Entity
@Table(name="ordini")
public class Ordini {

    @Id
    @GeneratedValue(strategy= GenerationType.AUTO)
    private int id;


    private int id_user;
    private int id_prodotti;
}
